packages/vaadin-checkbox/test/checkbox.test.js

```diff
@@ -43,11 +42,15 @@ describe('checkbox', () => {
     expect(checkbox.value).to.be.eql('on');
   });
 
-  it('fires click event', (done) => {
-    checkbox.addEventListener('click', () => {
-      done();
-    });
-    downAndUp(checkbox);
+  it('fires click event', () => {
+    const spy = sinon.spy();
+    checkbox.addEventListener('click', spy);
+
+    mousedown(checkbox);
```

Comment:
@web-padawan Not sure about this change if I did everything correctly. Although, I just reproduced the same behavior using helpers from the `@vaadin/testing-helpers` package.

Original [function](https://github.com/PolymerElements/iron-test-helpers/blob/8c6ad1d3fa3b73d38f4b63da785344c7ab99a5a6/mock-interactions.js):

```js
export function downAndUp(target, callback, options) {
  if (options && options.emulateTouch) {
    touchstart(target);
    touchend(target);
  }

  down(target);
  Base.async(function() {
    up(target);
    click(target);
    callback && callback();
  });
}
```

Reply:
The change seems good to me. I didn't copy the `downAndUp` function intentionally to avoid timing issues.

Reply:
Alright, thanks!

I also think that calling events explicitly is a more clear way that is a bit easier to understand in opposition to what is offered by Polymer helpers – `downAndUp`.